frappe/integrations/doctype/google_drive/google_drive.py

```diff
@@ -197,6 +197,7 @@ def upload_system_backup_to_google_drive():
 			file_urls.append(backup.backup_path_files)
 			file_urls.append(backup.backup_path_private_files)
 	else:
+		backup_files()
```

Comment:
just realized this would take files backups even if `account.file_backup` isn't set

Reply:
Adding condition, thanks for noticing.

Reply:
https://github.com/frappe/frappe/pull/11609